src/plugins/ini/README.md

```diff
@@ -134,7 +134,13 @@ key exists, the plugin treats lines starting with whitespaces as continuations o
 Keep in mind, that writing multiline values is also only supported if the multiline support is turned on.
 Otherwise the plugin will refuse to write key values with newlines.
 
+## METAKEY SUPPORT ##
 
+The ini plugin supports turning keys into metakeys (similar to the ni plugin) by adding the config key `meta`
+
+## PRESERVE ORDER ##
+
+If the `preserveorder` config key is set, the ini plugin preserves the original order of the file.
```

Comment:
I'd have to rewrite most testcases, and i currently can't guarantee that it works in all scenarios, thats why i wanted to keep it separated until i'm sure it doesn't break anything.
also, the algorithm is really ugly, but the only way i could realise it and i didn't want to force it on everyone the way it currently is.